packages/story-editor/src/components/panels/design/textStyle/stylePresets/useApplyStyle.js

```diff
@@ -30,10 +30,8 @@ function useApplyStyle({ pushUpdate }) {
   // Update size and position if relevant values have changed.
   usePresubmitHandler(getUpdatedSizeAndPosition, []);
 
-  const { selectedElements } = useStory(({ state: { selectedElements } }) => {
-    return {
-      selectedElements,
-    };
+  const selectedTextElements = useStory(({ state: { selectedElements } }) => {
+    return selectedElements?.filter(({ type }) => type === 'text');
```

Comment:
Q: If `selectedElements` is `undefined` here, would https://github.com/GoogleForCreators/web-stories-wp/pull/10672/files#diff-e37fde4c9173759792145832a2359437534a2fd118fa83f85160e7fd69a66cf1R106 error? Since it would be called with `undefined`.

Reply:
ah good catch. will add some behavior to account for that